docs/src/architecture/pallets/randomness.md

```diff
@@ -13,15 +13,18 @@
 
 ## Overview
 
-The randomness pallet saves a random seed for each block when it's finalized and allows retrieval of this randomness at a later time.
-There is a limitation - the randomness is available only after the 81st block of the chain, due to randomness predictability earlier.
-Currently, the seeds are used for the sealing pipeline's pre-commit and prove commit, in other words generating a replica and proving a sector.
+The Randomness Pallet provides secure randomness for on-chain operations, primarily for the proof systems used in storage verification.
```

Comment:
Not sure if it is truly secure, I would take it out and leave the next paragraph to explain